src/ray/common/id.h

```diff
@@ -395,7 +395,7 @@ std::ostream &operator<<(std::ostream &os, const PlacementGroupID &id);
    private:                                                                    \
     explicit type(const std::string &binary) {                                 \
       RAY_CHECK(binary.size() == Size() || binary.size() == 0)                 \
-          << "expected size is " << Size() << ", but got " << binary.size();   \
+          << "expected size is " << Size() << ", but got data " << binary << " of size " << binary.size();   \
```

Comment:
Why we should describe all data in RAY_CHECK?

Reply:
We need this information to know what the data is, which we may know where it is from

Reply:
We don't need that cause of nothing we can obtain from binary format in logging except converting it to hex.
The main reason is many of binary data start with b'0000', so it hard to read or record in log file. Besides, it also makes console printing messy codes.

Reply:
When check failed, the process will exit. We need enough information so that we can know what happened, For example, when trying fixing ci error, I  found out one of ID binary is the content of a task arg meta.